.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+AllCops:
+  TargetRubyVersion: 2.0.0
+
+Documentation:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Exclude:
+    - 'features/*'
+    - 'spec/**/*'
+
+Metrics/ClassLength:
+  Enabled: false
+
+Metrics/LineLength:
+  Enabled: false
```

Comment:
I like 80 because I can have three windows on my monitor and not have horizontal scrolling. I'm ok with 120 if it's overly burdensome, but I think most of the library is already 80

Reply:
Thanks y'all for the inputs. I went with 120 max, since I was encountering 150+ more offenses.